nixos/modules/services/x11/display-managers/ly.nix

```diff
@@ -0,0 +1,149 @@
+{ config, lib, pkgs, ... }:
```

Comment:
Since this is not x11 only, we should drop the x11 from the path similar to plasma6.